ddtrace/llmobs/_integrations/openai.py

```diff
@@ -148,7 +148,7 @@ def _llmobs_set_meta_tags_from_embedding(span: Span, kwargs: Dict[str, Any], res
         for doc in embedding_inputs:
             input_documents.append(Document(text=str(doc)))
         span._set_ctx_items({METADATA: metadata, INPUT_DOCUMENTS: input_documents})
-        if span.error:
+        if span.error or not resp:
```

Comment:
did the patching changes have an effect on llmobs embeddings spans (can embedding operations be streamed?? i could be behind on the times 😭), or was there a tangential issue?

Reply:
Nope, this is just a one-off change to just ensure we don't try tagging anything else if there's no resp!